.github/workflows/testing.yml

```diff
@@ -17,19 +17,20 @@ jobs:
     strategy:
       matrix:
         resolver:
-          - lts-17
```

Comment:
Dropped lts based on ghc 8.10.4.
I feel like nowadays it's reasonable to use ghc 8.10.7 as the oldest "tested" version.